translate.js

```diff
@@ -0,0 +1,115 @@
+function translateErrors (ret, errors) {
+  for (var error in errors) {
+    var type = 'error'
+    var extractType = /^(.*):(\d+):(\d+):(.*):/
+    extractType = extractType.exec(errors[error])
+    if (extractType) {
+      type = extractType[4].trim().toLowerCase()
+    } else if (errors[error].toLowerCase().indexOf(': warning:')) {
+      type = 'warning'
+    } else if (errors[error].toLowerCase().indexOf(': error:')) {
+      type = 'error'
+    }
+    ret.push({
+      type: 'Error',
+      component: 'general',
+      severity: type,
```

Comment:
@yann300 `severity` must be either `'warning'` or `'error'`, while `type` should be the readable type.